lib/blynk/BlynkLib.py

```diff
@@ -355,6 +360,10 @@ def tweet(self, msg):
     def email(self, to, subject, body):
         if self.state == AUTHENTICATED:
             self._send(self._format_msg(MSG_EMAIL, to, subject, body))
+            
+    def virtualWrite(self, pin, val): # TODO: *args?

```

Comment:
fixed.
@danicampora , yes this is supported by the protocol. (when you receive write handler, there is also a list of args).
However, we don't need to support it immediately.